pkg/network/protocols/core/protocol.go

```diff
@@ -216,7 +216,7 @@ func (p *Protocol) onBlock(blockData []byte, id peer.ID) {
 
 func (p *Protocol) onBlockRequest(idBytes []byte, id peer.ID) {
 	if len(idBytes) != iotago.BlockIDLength {
-		p.Events.Error.Trigger(ierrors.Wrap(iotago.ErrInvalidIdentifierLength, "failed to deserialize block request"), id)
+		p.Events.Error.Trigger(ierrors.Errorf("failed to deserialize block request: invalid block id length"), id)
```

Comment:
Isn't there the `%s` missing?
Or do you want to use `ierrors.New` instead?

Reply:
The `id` is passed to `Trigger` not to `Errorf`.